onboarding/source/controller.c

```diff
@@ -97,16 +116,24 @@ static void controllerTask(void * pvParameters) {
     obc_error_code_t lightServiceStatus = initLightService();
     if (lightServiceStatus != OBC_ERR_CODE_SUCCESS) {
         /* USER CODE BEGIN */
-        // Deal with error when initializing light service task and/or queue
+        char str[] = "Error: Light service was not properly initialized";
+        sciPrintText((unsigned char *) str, strlen(str));
 
-        /* USER CODE END */
     } else { 
-        /* Light service task and queue created successfully */
+        /* USER CODE END 
+ 
+        Light service task and queue created successfully */
         BaseType_t xReturned; 
         xReturned = xTimerStart(ledTimerHandle, 0);
         
         /* USER CODE BEGIN */
-        // Start light timer and check if both timers were started successfully
+        xReturned = xTimerStart(lightTimerHandle, 0);
+
+        if (xReturned == pdFAIL) {
+             char str[] = "Error: Light Timer was not properly initialized";
+             sciPrintText((unsigned char *) str, strlen(str));
```

Comment:
Note that there's a `sciPrintf` function that behaves very similarly to the stdio `printf` function. It's cleaner to use that instead.